src/tests/functional/plugin/meta/auto_batch/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME ov_auto_batch_func_tests)
```

Comment:
all tests should be located in plugin's folder.
for autobatch it's src/plugins/auto_batch/tests

Reply:
ok